src/html/extractors/factories/embeddedAttrJs.ts

```diff
@@ -0,0 +1,31 @@
+import { JsParser } from '../../../js/parser';
+import { Validate } from '../../../utils/validate';
+import { Element, IHtmlExtractorFunction, Node } from '../../parser';
+/**
+ * embeddedAttrJsExtractor extractor messages from element attribute,
+ * but treat them as embedded js:
```

Comment:
Description is inaccurate as it doesn't extract any messages. It just runs the given js parser on any matching attribute values.